evap/staff/management/commands/run_tasks.py

```diff
@@ -26,12 +29,23 @@ def update_courses(self):
                     course.evaluation_end()
                     if course.is_fully_reviewed():
                         course.review_finished()
+                        if not course.is_graded:
+                            course.publish()
+                            evaluation_results_courses.append(course)
+                        elif course.final_grade_documents.exists():
+                            course.publish()
+                            grade_document_courses.append(course)
+                            evaluation_results_courses.append(course)
+                    elif course.final_grade_documents.exists():
+                        grade_document_courses.append(course)
```

Comment:
i'm not quite sure whether it's cool to have this much code in a management command. i mean, it's an integral part of a course's life cycle that is in a completely different place. if you don't disagree, please move as much code as possible into the Course class.